clients/rospy/src/rospy/core.py

```diff
@@ -145,7 +145,22 @@ def rospywarn(msg, *args):
     _rospy_logger.warn(msg, *args)
 
 
-def _base_logger(msg, name=None, throttle=None, level='info'):
+def _base_logger(msg, *args, **kwargs):
+
+    try:
+        name = kwargs.pop('_logger_name')
+    except KeyError:
+        name = None
+
+    try:
+        throttle = int(kwargs.pop('_logger_throttle'))

```

Comment:
I thought the throttle argument is expecting an int, but it doesn't look like that's the case, so it's probably not needed.